README.md

```diff
@@ -28,6 +28,7 @@ Usage: migrate-mongo [options] [command]
 
   Options:
 
+    -c, --custom   specify a custom migration file (only for up)
```

Comment:
@bdcorps having a down custom migration also will be super useful. Specially for some quick hot fixes. See if you can consider a down migration support for custom migration.